spec/services/work_packages/update_service_integration_spec.rb

```diff
@@ -773,7 +773,7 @@
     # rubocop:enable RSpec/MultipleExpectations
   end
 
-  describe 'rescheduling work packages forward follows/hierarchy relations' do
+  describe 'rescheduling work packages backwards with follows/hierarchy relations' do
```

Comment:
I don't think this spec does a lot any more. Since the `following_work_package` isn't moved, non of the others are moved. 

Either we remove the spec altogether or we simplify it at the least. I'd opt for removing it.

Reply:
Thanks! I hesitated as it was really testing lots of relations.

I'll remove it.